src/topojson/topojson.ml

```diff
@@ -1 +1,16 @@
 (* Implentation of TopoJSON Objects *)
+
+module Geometry = struct
+  module LineString = struct
```

Comment:
This is heading in the right direction. I've spent a bit of time having a closer look at the specification and I'm not convinced anymore that we will be able to share that much with `Geojson` because the underlying geometries change quite a bit (and therefore so do the functions for getting things like `foreign_members`). In particular, for `LineString` it can only contain "arcs" now and not "coordinates". So looks like we'll have to reimplement a few bits copying from Geojson.

This is the general structure I was thinking after reading the spec. What do you think?

```ocaml
module Make (G : Geojson.S) = struct
  module Geometry = struct

    module Point = G.Geometry.Point
    module LineString = struct
      type t = int array
      let v arcs = arcs
    end

    type geometry =
      | Point of Point.t
      | LineString of LineString.t
    
    and t = { geometry : geometry; id : G.json option; properties : (string * G.json) list; foreign_members : (string * G.json) list; }
  end

  module Topology = struct
    type t
  end

  type t = Topology of Topology.t | Geometry of Geometry.t
end
```

Note I've functorised over some implementation of `Geojson`. I'm also copying a little bit from https://github.com/georust/topojson.